proto/beacon/p2p/v1/types.proto

```diff
@@ -95,13 +95,14 @@ message AttestationData {
 
 message ValidatorRecord {
   bytes pubkey = 1;
-  uint64 withdrawal_shard = 2;
-  bytes withdrawal_address = 3;
-  bytes randao_commitment = 4;
+  // TODO(781): The usage of withdrawal_credentials is not defined in spec. Not used in Prysm yet.
+  bytes withdrawal_credentials = 2;
+  bytes randao_commitment = 3;
+  uint64 randao_skips = 4;
   uint64 balance = 5;
   uint64 status = 6;
```

Comment:
Do we think this is going to be an enum? If not, please comment that this is a status code of some kind. 

If possible, link to the list of possible status codes.